README.rst

```diff
@@ -68,6 +68,27 @@ Add a `pyproject.toml` file on the root of your project with the following conte
 
 z3c.dependencychecker will read this information and use it on its reports.
 
+Ignore packages
+---------------
+
+Sometimes you need to add a package on `setup.py` although you are not importing it directly,
+but maybe is an extra dependency of one of your dependencies,
+or your package has a soft dependency on a package,
+and as a soft dependency it is not mandatory to install it always.
+
+`z3c.dependencychecker` would complain in both cases.
+It would report that a dependency is not needed,
+or that a missing package is not listed on the package requirements.
+
+Fortunately, `z3c.dependencychecker` also has a solution for it.
+
+Add a `pyproject.toml` file on the root of your project with the following content:
+
+    [tool.dependencychecker]
+    ignore-packages = ['one-package', 'another.package' ]
+
+`z3c.dependencychecker` will read this information and use it on its reports.
```

Comment:
`z3c.dependencychecker` will totally ignore those packages in its reports, whether they're requirements that appear to be unused or requirements that appear to be missing.

Reply:
Changed